bundle/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Instrument.scala

```diff
@@ -0,0 +1,13 @@
+package edu.gemini.seqexec.server
+
+import edu.gemini.spModel.config2.Config
+
+import scala.concurrent.Future
+
+/**
+ * Created by jluhrs on 4/22/15.
+ */
+trait Instrument extends System {
+  def observe(config: Config): Future[Unit]
```

Comment:
Yes, I think it will make sense, but the time to dwell on those details has yet to come.